pkg/app/app.go

```diff
@@ -4,6 +4,7 @@
 package app
 
 import (
+	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/componentInfo"
```

Comment:
this behaviour of GoLand is annoying.. ignore the import formatting, I will go fix it later before proper review